evm_arithmetization/src/cpu/kernel/asm/mpt/read.asm

```diff
@@ -121,7 +121,7 @@ mpt_read_extension_found:
     // stack: child_ptr, future_nibbles, key, retdest
     %jump(mpt_read) // recurse
 
-mpt_read_leaf:
+global mpt_read_leaf:
```

Comment:
The `global` label changes on `mpt/insert.asm` and `mpt/read.asm` are unrelated to this PR, but I got extremely confused when debugging through the Kernel logs when seeing `mpt_read_leaf_not_found`, while it actually was, because all these labels are below their global negative counterpart.
Hence I'd think this is better debugging wise if they were also made global, to prevent any confusion.